tests/testing/unit/support/selector.ts

```diff
@@ -70,7 +70,7 @@ describe('selector', () => {
 			assert.deepEqual(adapter.getChildren(v('div', {}, [w(WidgetBase, {})])), [w(WidgetBase, {})]);
 			assert.deepEqual(adapter.getChildren(w(WidgetBase, {}, [v('span')])), [v('span')]);
 			assert.deepEqual(adapter.getChildren(w(WidgetBase, {}, [w(WidgetBase, {})])), [w(WidgetBase, {})]);
-			assert.deepEqual(adapter.getChildren(w(WidgetBase, {})), []);
+			assert.deepEqual(adapter.getChildren(w(WidgetBase, {})), undefined);
```

Comment:
how come this had to change?

Reply:
Because by default children are not set to an empty array when using `w` any more.

Reply:
but this is the assertion for `getChildren` no? which is both typed as an array and returns an empty array if not set?

Reply:
If it is a WNode or VNode then it just returns the children from the element.

Reply:
Ah okay, sorry I missed that this was in the selector, and not the assertion template. This makes me concerned about these types/values a little bit though. 

I think any kind of `children` interface should be consistently returning either an empty array `[]` for no children, an array of one `['a']` for one item, and an array of many `['a','b','c']` for many items. This means the end user can consistently work on an array without guards for api's such as this, and would match other child like api's such as `NodeList`.

I'm not sure what the ramifications are on types or whether it's possible with `TSX`, but it'd seem more consistent to make the user define `create().children<[ whatever ]>()` over `create().children<whatever>()` if they need to define a single item.

Reply:
I think that's fair point about always having children and it being an array (that can be defaulted). I've made some changes (WIP atm as need to put back the defaulting of children) that added support for children to be mandatory and able to be specified singularly but are consumed by the widget author as the first item in an array.